rest/models/contracts.go

```diff
@@ -0,0 +1,161 @@
+package models
+
+// GetOptionsContract is the set of parameters for the GetOptionsContract method.
+type GetOptionsContractParams struct {
+	// Return the contract that contains this options ticker.
+	OptionsTicker string `validate:"required" path:"options_ticker"`
```

Comment:
I think we could change this to follow our typical pattern without losing clarity:
```
Ticker string `validate:"required" path:"ticker"`
```

Reply:
Changed OptionsTicker to Ticker, but the path is options_ticker so I left that as is. 
/v3/reference/options/contracts/{options_ticker}